generators/entity/index.js

```diff
@@ -238,6 +238,38 @@ module.exports = EntityGenerator.extend({
             this.skipServer = true;
         }
     },
+
+    _loadOtherEntityJson: function(entityName) {
```

Comment:
As I said we already read it for tableName somewhere hence please try to reuse that or extarct both into this common function so that we dont read the same file twice. If you cant find where its read let me know I'll point you to there as I need to look it up as well. But if im not wrong its done in the liquibase changelog template I guess